core/src/data_sources/qdrant.rs

```diff
@@ -555,4 +580,8 @@ impl DustQdrantClient {
             }
         }
     }
+
+    pub fn raw_client(&self) -> Arc<QdrantClient> {
```

Comment:
@spolu, since https://github.com/dust-tt/dust/pull/5148, I haven't been able to access the Qdrant client. I didn't want to expose all those methods in our client, so I created this method to gain access to the raw client.